airbase/pom.xml

```diff
@@ -162,33 +162,32 @@
 
         <!-- Plugin versions used in multiple places -->
         <dep.plugin.scm.version>2.0.1</dep.plugin.scm.version>
-        <dep.plugin.surefire.version>3.2.1</dep.plugin.surefire.version>
+        <dep.plugin.surefire.version>3.2.2</dep.plugin.surefire.version>
         <dep.plugin.pmd-runtime.version>6.55.0</dep.plugin.pmd-runtime.version>
 
         <!-- Packaging for the tarball deployment -->
         <dep.packaging.version>0.163</dep.packaging.version>
 
         <!-- Dependency versions that should be the same everywhere. -->
         <dep.assertj-core.version>3.24.2</dep.assertj-core.version>
-        <dep.bval.version>2.0.6</dep.bval.version>
         <dep.guava.version>32.1.3-jre</dep.guava.version>
         <dep.guice.version>7.0.0</dep.guice.version>
         <dep.jackson.version>2.15.3</dep.jackson.version>
         <dep.jmh.version>1.37</dep.jmh.version>
         <dep.jmxutils.version>1.24</dep.jmxutils.version>
         <dep.joda.version>2.12.5</dep.joda.version>
-        <dep.junit.version>5.10.0</dep.junit.version>
-        <dep.kotlin.version>1.9.0</dep.kotlin.version>
+        <dep.junit.version>5.10.1</dep.junit.version>
+        <dep.kotlin.version>1.9.21</dep.kotlin.version>
         <dep.logback.version>1.4.8</dep.logback.version>
```

Comment:
@wendigo can you also update this to 1.4.14? It fixes two CVEs: https://logback.qos.ch/news.html

Reply:
No. It's broken, hangs the execution in Trino.

Reply:
Already tested. There is a regressionm in 1.4.9 lurking somewhere

Reply:
Does it make sense to test 1.3.14?

Reply:
Already did that. Still failing. You can try it on your own

Reply:
It requires changes in Airlift: https://github.com/airlift/airlift/pull/1112